core/assets/bundles/bundle.properties

```diff
@@ -597,6 +597,7 @@ setting.crashreport.name = Send Anonymous Crash Reports
 setting.savecreate.name = Auto-Create Saves
 setting.publichost.name = Public Game Visibility
 setting.chatopacity.name = Chat Opacity
+setting.lasersopacity.name = Power Lasers Opacity
```

Comment:
I would call this "Power *Laser* Opacity".

Reply:
Done.